### PR TITLE
WMS #8021: Oracle AQ 

### PR DESCRIPTION
**Pull request by matayal**

### CODE REVIEW

File: oracleAQ/java/.gitignore [hunk 1]
```diff
@@ -0,0 +1,33 @@
+HELP.md
```
Comment (paulparkinson) [inline oracleAQ/java/.gitignore:1]: please remove this file

File: oracleAQ/java/.mvn/wrapper/MavenWrapperDownloader.java [hunk 2]
```diff
@@ -0,0 +1,117 @@
+/*
```
Comment (paulparkinson) [inline oracleAQ/java/.mvn/wrapper/MavenWrapperDownloader.java:1]: please remove this .mvn dir

File: oracleAQ/java/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,6 @@
+mvn clean install
```
Comment (paulparkinson) [inline oracleAQ/java/Dockerfile:1]: for Java builds we've been building and then packaging in dockerfile. this prevents building every time the service is deployed and also allows us to change the JDK image to jre-slim. see inventory-helidon etc. for examples

File: oracleAQ/java/src/main/java/com/aq/java/OracleAQController.java [hunk 4]
```diff
@@ -0,0 +1,64 @@
+package com.aq.java;
```
Comment (paulparkinson) [inline oracleAQ/java/src/main/java/com/aq/java/OracleAQController.java:1]: naming the directory "java" makes things confusing/ambiguous with the "main/java" src location. please rename it to "aq-java" or some such

**Review by paulparkinson — CHANGES_REQUESTED** [review]
Made some comments inline. I have not reviewed the actual src detail yet just the basics re structure, etc..  In addition...
- I would rename the root dir from "oracleAQ" to the name of the app such as "cloudbank" or whatever it is to follow the convention of "Grabdish"
- It would be good to make the db etc. infra setup use the common core that @RichardExley has been working on so let's get him to review that aspect as well.
- Also, need to add copyright at the top of all files
Thanks.

**Comment by matayal** [comment]
refactoring the location of code.

**Pull request closed without merging**